cli/lsp/capabilities.rs

```diff
@@ -154,5 +154,8 @@ pub fn server_capabilities(
     // TODO(nayeemrmn): Support pull-based diagnostics.
     diagnostic_provider: None,
     inline_value_provider: None,
+    inline_completion_provider: None,
+    // TODO(nayeemrmn); Provide!
```

Comment:
I assume this is gonna happen in the follow up PR?

Reply:
Yeah, though I hadn't decided that when writing that. Removed the comment.